### PR TITLE
chore(client): changing credits to use a markdown file and generalizating style

### PR DESCRIPTION
**Pull request by Dracks**

### CODE REVIEW

File: client/levels/menu.tscn [hunk 1]
```diff
@@ -127,6 +135,15 @@ func _on_button_back_pressed() -> void:
 func _ready() -> void:
 	if not Engine.is_editor_hint():
 		hide()
+	else:
+		update_credits()
+
+func update_credits() -> void:
```
Comment (Dracks) [inline client/levels/menu.tscn:141]: this loads the file when the editor is open, **It doesn't update it on cli** which means the pipeline is wrong.

File: client/levels/menu.tscn [hunk 2]
```diff
@@ -756,107 +826,172 @@ custom_minimum_size = Vector2(455, 500)
...
-[center][img]res://godot_icon.svg[/img][/center]
+[img]res://godot_icon.svg[/img]
 
 
-[url=https://github.com/Codename-Recon/Codename-Recon/blob/master/LICENSE_ASSETS.md]Asset License[/url]"
-script = SubResource("GDScript_xjdvj")
+[url=https://github.com/Codename-Recon/Codename-Recon/blob/master/LICENSE_ASSETS.md]Asset License[/url]
+"
+script = ExtResource("19_8a2oa")
+markdown_text = "# Credits
```
Comment (Dracks) [inline client/levels/menu.tscn:914]: with this aproach library, we get the credits by duplicated, as it saves the text variable of the richtext, is something I don't like it too much. 

A proposal will be, that we have a softlink (I don't know if git handles it), or similar stuff, and have the markdown as a file in the project (**included in the exports too**) and we simply load that on the menu start or similar. What do you think?

Comment (HunterNN) [reply client/levels/menu.tscn:914]: Sounds good. You could place it under `client/assets/text/`. Keep in mind to update this field for every export variation to include Markdown files: 
image

**Review by HunterNN — COMMENTED** [review]
Since you implemented outline color and size in the default theme, you can deactivate it on the node itself. Otherwise it overwrites the default values (outline size = 5 instead of 7)

image

**Review by HunterNN — CHANGES_REQUESTED** [review]
Since you implemented outline color and size in the default theme, you can deactivate it on the node itself. Otherwise it overwrites the default values (outline size = 5 instead of 7)

image

**Comment by Dracks** [comment]
> Since you implemented outline color and size in the default theme, you can deactivate it on the node itself. Otherwise it overwrites the default values (outline size = 5 instead of 7)
> 
> image

weird, as originally I dropped it, but now It will be dropped.

**Review by HunterNN — APPROVED** [review]

**Pull request merged**